docs/conf.py

```diff
@@ -26,25 +26,11 @@
     import tomllib
 
 
-def find_mod_objs_patched(*args, **kwargs):
-    from sphinx_automodapi.utils import find_mod_objs
-
-    return find_mod_objs(args[0], onlylocals=True)
-
-
-def patch_automodapi(app):
-    """Monkey-patch the automodapi extension to exclude imported members"""
-    from sphinx_automodapi import automodsumm
-
-    automodsumm.find_mod_objs = find_mod_objs_patched
-
-
```

Comment:
Monkey patches like this are generally a bad idea. The whole reason for the need of this patch was poor construction of `stuserdict.py` module to begin with. https://github.com/spacetelescope/roman_datamodels/pull/156#issuecomment-1520536988 details a couple of different ways of properly resolving this without monkey patching. However, all of those issues are moot because this PR removes `stuserdict.py`.